inst/tinytest/test_box.R

```diff
@@ -0,0 +1,6 @@
+library(classInt)
+set.seed(101)
+x <- rnorm(50)
+expect_error(print(classIntervals(x, style="box", iqr_mult=1.5, type=5, legacy=TRUE)))
+expect_inherits(classIntervals(x, style="box", iqr_mult=1.5, type=5), "classIntervals")
+
```

Comment:
If we enforce  a maximum 0 value for `iqr_mult` we can add an additional test (currently failing) as:

```r
expect_inherits(print(classIntervals(x, style="box", iqr_mult=-1, type=5)), "table")
```

Reply:
I'd rather use a `stopifnot()` than silently change an invalid value. Would it be better to  tell the user that negative (possibly non-positive) values are invalid?